### DISCUSSION TITLE
I can't find the namespace belong to what appId

### DISCUSSION THREAD

**Discussion by CATTechnology** [apolloconfig/apollo#4626] [Q&A] [answered]
问题描述：项目里面引用了appId下的namespace，但是在项目下找不到该namespace
原因：公司内部项目存在继承关系，如果因为之前的开发者引用了某个父级namespace，并且没有说明，会导致后续的开发者没办法找到该namespace的正确位置，并且在源码里面也没有改namespace的元数据信息
client调用得接口如下：http://test-apollo-meta.xxx.com/configs/{appId}/default/{namespace}?ip=${ip}

需求点：我应该如何在不确定namespace位置的情况下，定位到这个namespace是在哪个父级项目里面

**Comment by nobodyiam** [comment]
You may upgrade to the 2.0.x version to use the public namespace search feature.

![image](https://user-images.githubusercontent.com/837658/200108486-145d4863-721f-48c1-8ec0-a5e0b97d768c.png)

**Reply by CATTechnology** [reply]
这个功能在低版本有对应的入口（api）么

**Comment by CATTechnology** [comment, marked as answer]
通过http://apollo-portal.xxx.com/appnamespaces/public 这个接口可以获取到所有的公共namespace
